ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly-netobserv.yaml

```diff
@@ -0,0 +1,60 @@
+base_images:
```

Comment:
Instead of creating new file, how about append the tests into *__amd64-nightly.yaml ? You can rename your test from `aws-ipi-f7` to `aws-ipi-f7-netobserv` to indicate it's specific for netobserv

Reply:
@liangxia I guess I could, but I did not want to clutter the *__amd64-nightly.yaml file since I thought it would typically have jobs where payload tests are run. Also, I largely modeled this around logging jobs, would you prefer to have it in `*__amd64-nightly.yaml` file?

Reply:
We already have 150+ config files in this folder. I can't imagine if every subteam add a separate config file for each version for their team.